### PR TITLE
circleci can deploy to test environment

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -58,6 +64,54 @@ jobs:
...
+          name: kubectl configure context
+          command: 'kubectl config set-context "circleci_test_production" --cluster="$KUBE_CLUSTER" --user="circleci_test_production" --namespace="formbuilder-platform-test-production"'
+      - run:
+          name: kubectl use circleci context
+          command: 'kubectl config use-context circleci_test_production'
+      - run:
+          name: apply kubernetes changes to test production
+          command: './scripts/deploy_platform.sh -p test -d production'
+      - run:
+          name: delete pods in test production
```
Comment (solidgoldpig) [inline .circleci/config.yml]: Presumably this bit can be retired when we have rolling updates in place?

File: package.json [hunk 2]
```diff
@@ -18,6 +18,6 @@
   "homepage": "https://github.com/ministryofjustice/fb-user-datastore#readme",
   "dependencies": {},
   "devDependencies": {
-    "@ministryofjustice/fb-deploy-utils": "1.0.5"
+    "@ministryofjustice/fb-deploy-utils": "^1.0.6"
```
Comment (solidgoldpig) [inline package.json]: Why not latest?

File: .circleci/config.yml (hunk 1 above)
Comment (asmega) [reply .circleci/config.yml]: yes

File: package.json (hunk 2 above)
Comment (asmega) [reply package.json]: 1.0.6 is the latest

**Pull request merged**